sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -1291,36 +1320,25 @@ export interface ShingleTokenFilter {
 }
 
 // @public
-export type SimpleDataType = "Edm.String" | "Edm.Int32" | "Edm.Int64" | "Edm.Double" | "Edm.Boolean" | "Edm.DateTimeOffset" | "Edm.GeographyPoint" | "Collection(Edm.String)" | "Collection(Edm.Int32)" | "Collection(Edm.Int64)" | "Collection(Edm.Double)" | "Collection(Edm.Boolean)" | "Collection(Edm.DateTimeOffset)" | "Collection(Edm.GeographyPoint)";
+export type SimpleDataType = "Edm.String" | "Edm.Int32" | "Edm.Int64" | "Edm.Double" | "Edm.Boolean" | "Edm.DateTimeOffset" | "Edm.GeographyPoint";
```

Comment:
same question, confused why we lost all the collection variants

Reply:
This is the result of the latest PR: https://github.com/Azure/azure-rest-api-specs/pull/9301/files#diff-a01e10da9dba0149960db35a9d106976L2632. As a result the collection is gone from the generated code. As a result, I removed it from the custom code too